packages/protocol/test-sol/unit/common/GoldToken.t.sol

```diff
@@ -1,54 +1,71 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.5.13;
 
-import "celo-foundry/Test.sol";
 import "@celo-contracts/common/GoldToken.sol";
 import "@test-sol/unit/common/GoldTokenMock.sol";
 
-import { TestConstants } from "@test-sol/constants.sol";
+import { Utils } from "@test-sol/utils.sol";
+import "@test-sol/utils/WhenL2.sol";
 
-contract GoldTokenTest is Test, TestConstants, IsL2Check {
+contract GoldTokenTest is Utils {
```

Comment:
ditto as here https://github.com/celo-org/celo-monorepo/pull/11272/files#r1848861672

Reply:
will make these changes in a separate PR.

[link](https://github.com/celo-org/celo-blockchain-planning/issues/766) to tracking issue